k8s/agones/Makefile

```diff
@@ -0,0 +1,97 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= 0.8.1
+DEMO_RELEASE_TAG ?= release-0.8.0
+NAME ?= agones
+
+
+$(info ---- TAG = $(TAG))
+
+
+# source repo and destination tags for images
+SOURCE_REGISTRY ?= gcr.io/agones-images
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/agones/deployer:$(TAG)
+AGONES_CONTROLLER_IMAGE ?= $(REGISTRY)/agones:$(TAG)
+AGONES_SDK_IMAGE ?= $(REGISTRY)/agones/agones-sdk:$(TAG)
+AGONES_PING_IMAGE ?= $(REGISTRY)/agones/agones-ping:$(TAG)
+
+
+APP_PARAMETERS ?= { \
+    "APP_INSTANCE_NAME": "$(NAME)", \
+    "NAMESPACE": "$(NAMESPACE)" \
+}
+
+
+APP_TEST_PARAMETERS ?= { \
+}
+
+
+app/build:: .build/agones/deployer \
+            .build/agones/agones-controller \
+            .build/agones/agones-sdk \
+            .build/agones/agones-ping
+
+
+.build/agones: | .build
+	mkdir -p "$@"
+
+
+app/upstream-chart: chart/agones-mp/requirements.* \
+                    | .build/agones
+	(helm repo list | grep "https://agones.dev/chart/stable") || \
+	    helm repo add agones "https://agones.dev/chart/stable"
+	helm dependency build chart/agones-mp
```

Comment:
If you simply add without checking, because repository label/key/identifier can be different from what the user used previously, you might end with a second instance of the same URL in Helm. By check for the existing of the URL you guarantee that it won't happen.

Reply:
Let's remove this from the Makefile since we don't want to confuse anyone with it. Upgrading versions is not as simple as updating the chart.